lib/applicants/get-applicant.ts

```diff
@@ -0,0 +1,46 @@
+import { gql } from '@apollo/client'; // gql tag
+import { Applicant } from '@lib/graphql/types'; // GraphQL types
+
+/**
+ * GQL query to fetch employees based on filter
+ */
+export const GET_APPLICANT_QUERY = gql`
+  query getApplicant($id: ID!) {
+    applicant(id: $id) {
+      firstName
+      lastName
+      email
+      phone
+      addressLine1
+      addressLine2
+      city
+      postalCode
+      id
+      rcdUserId
+    }
```

Comment:
Let's also add autofill for shipping address (not billing address). Need to also select shipping address fields